### PR TITLE
INT-234: Fixed position for Japan homepage nav on desktop

### PR DESCRIPTION
**Pull request by perjg**
Global nav should be fixed to top of screen in desktop mode.

### CODE REVIEW

File: homepage/front/styles/component/_nav.scss [hunk 1]
```diff
@@ -25,6 +25,10 @@ header.navigation {
 	width: 100%;
 	z-index: 999;
 
+	@include media ($horizontal-bar-mode) {
```
Comment (RafLeszczynski) [inline homepage/front/styles/component/_nav.scss]: Please put `@include` before declaring local properties for given selector

Comment (perjg) [inline homepage/front/styles/component/_nav.scss]: @RafLeszczynski This seems to be in conflict with the rules in styleguide/.scss-lint.yml 

If I move the include up to right under header.navigation I get this error:

homepage/front/styles/component/_nav.scss:26 [W] DeclarationOrder: Expected item on line 26 to appear before line 22. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets

Comment (perjg) [inline homepage/front/styles/component/_nav.scss]: so according to this, @include with no properties should come before properties, but @include with content should go after.

Comment (RafLeszczynski) [inline homepage/front/styles/component/_nav.scss]: You're right, my mistake

**Comment by perjg** [comment]
@RafLeszczynski Thanks for clarifying.

Unless there are other issues, I will merge this tomorrow (Friday)

**Pull request merged**